packages/fiori/src/NotificationListItemBase.ts

```diff
@@ -0,0 +1,277 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import { isSpace } from "@ui5/webcomponents-base/dist/Keys.js";
+import { getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import slot from "@ui5/webcomponents-base/dist/decorators/slot.js";
+import event from "@ui5/webcomponents-base/dist/decorators/event.js";
+import type I18nBundle from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import { getEventMark } from "@ui5/webcomponents-base/dist/MarkedEvents.js";
+import ListItemBase from "@ui5/webcomponents/dist/ListItemBase.js";
+import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
+import Priority from "@ui5/webcomponents/dist/types/Priority.js";
+import Popover from "@ui5/webcomponents/dist/Popover.js";
+
+// Icons
+import "@ui5/webcomponents-icons/dist/decline.js";
+import "@ui5/webcomponents-icons/dist/message-success.js";
+import "@ui5/webcomponents-icons/dist/message-error.js";
+import "@ui5/webcomponents-icons/dist/message-warning.js";
+import "@ui5/webcomponents-icons/dist/overflow.js";
+
+// Templates
+import NotificationOverflowActionsPopoverTemplate from "./generated/templates/NotificationOverflowActionsPopoverTemplate.lit.js";
+
+// Styles
+import NotificationOverflowActionsPopoverCss from "./generated/themes/NotificationOverflowActionsPopover.css.js";
+
+/**
+ * Defines the icons corresponding to the notification's priority.
+ */
+const ICON_PER_PRIORITY = {
+	[Priority.High]: "message-error",
+	[Priority.Medium]: "message-warning",
+	[Priority.Low]: "message-success",
+	[Priority.None]: "",
+};
+
+type NotificationActionInfo = {
+	icon: string;
+	text: string;
+	_id: string;
+	disabled?: boolean;
+	design?: string;
+};
+
+/**
+ * @class
+ *
+ * The base class of the <code>NotificationListItem</code> and <code>NotificationListGroupItem</code>.
+ *
+ * @abstract
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webc.fiori.NotificationListItemBase
+ * @extends sap.ui.webc.main.ListItemBase
+ * @tagname ui5-li-notification-group
+ * @since 1.0.0-rc.8
+ * @appenddocs sap.ui.webc.fiori.NotificationAction
+ * @public
+ */
+
+/**
+ * Fired when the <code>Close</code> button is pressed.
+ *
+ * @event sap.ui.webc.fiori.NotificationListItemBase#close
+ * @param {HTMLElement} item the closed item.
+ * @public
+ */
+@event("close", {
```

Comment:
We [agreed](https://github.com/SAP/ui5-webcomponents/blob/main/docs/5-development/07-ts.md#events) to add event detail